TMDbLib/Objects/Search/SearchCollection.cs

```diff
@@ -4,6 +4,9 @@ namespace TMDbLib.Objects.Search
 {
     public class SearchCollection
     {
+        [JsonProperty("adult")]
+        public bool Adult { get; set; }
```

Comment:
Should this be nullable? Or is it always returned

Reply:
It's always available, false or true